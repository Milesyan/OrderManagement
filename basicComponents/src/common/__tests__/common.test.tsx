import Divider from '../Divider';
import TestRenderer from 'react-test-renderer';

test('adds 1 + 2 to equal 3', () => {
  const ele = TestRenderer.create(<Divider/>)
  expect(ele).toMatchSnapshot()
});