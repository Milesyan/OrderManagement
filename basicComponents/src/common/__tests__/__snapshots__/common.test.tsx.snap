// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render common components 1`] = `
<div
  className="divider"
/>
`;
